openwisp_users/tests/test_models.py

```diff
@@ -43,6 +44,19 @@ def test_create_user_without_email(self):
         u.save()
         self.assertIsNone(u.email)
 
+    def test_create_user_email_is_lowered(self):
+        self._create_user(username='user1', email='USER@example.com')
+        with self.subTest("Test that email is lowered"):
+            user = User.objects.get(username='user1')
+            self.assertEqual('user@example.com', user.email)
+        with self.subTest(
+            "Test creation of new user with the same mail but different case"
+        ):
+            u = User(username='user2', email='uSeR@example.com')
+            with self.assertRaises(Exception) as raised:
+                u.save()
+            self.assertEqual(IntegrityError, type(raised.exception))
```

Comment:
we don't want to fail with IntegrityError but with ValidationError, which is catched by the django admin and shown to the user in a comprehensible way (IntegrityError would fail with a generic 500 internal server error).